tests/test_vav_minimum_turndown_during_reheat.py

```diff
@@ -0,0 +1,148 @@
+import sys
+import unittest
+
+sys.path.append("./constrain")
+import datetime
+
+import pandas as pd
+from lib_unit_test_runner import *
+
+
+class TestVAVTurndown(unittest.TestCase):
+    def test_vav_turndown_untested(self):
+        points = [
+            "reheat_coil_flag",
+            "V_dot_VAV",
+            "V_dot_VAV_max",
+            "VAV_min_turndown_design",
+            "turndown_tol",
+        ]
+
+        timestamp = [
+            datetime(2024, 8, 1, 12, 0, 0),
+            datetime(2024, 8, 1, 13, 0, 0),
+        ]
+
+        data = [
+            [False, 0.005, 0.01, 0.3, 0.01],
+            [False, 0.005, 0.01, 0.3, 0.01],
+        ]
+
+        df = pd.DataFrame(data, columns=points, index=timestamp)
+
+        verification_obj = run_test_verification_with_data(
+            "VAVMinimumTurndownDuringReheat", df
+        )
+        results = list(verification_obj.result)
+        expected_results = [
+            "Untested",
+            "Untested",
+        ]
+
+        self.assertEqual(results, expected_results)
```

Comment:
check binary result here to be untested

Reply:
I'm not sure if I understood this comment correctly. Would @leijerry888 like me to modify the code or to check if this unit test passes after deleting the ```check_bool``` method (above comment)?

Reply:
Would this work?

```python
self.assertEqual(verification_obj.check_bool(), "Untested")
```

Reply:
Yes! This works!